src/main/java/com/amazon/opendistroforelasticsearch/security/securityconf/impl/v7/ConfigV7.java

```diff
@@ -112,6 +113,7 @@ public String toString() {
         public String filtered_alias_mode = "warn";
         public boolean disable_rest_auth;
         public boolean disable_intertransport_auth;
+        public boolean passive_intertransport_auth;
```

Comment:
What if this flag is enabled on a domain with prior versions of the security plugin installed? Won't it cause backward compatibility issue and failures due to jackson unknown property exception?

Reply:
Current design does not support backward compatibility when introducing new feature flags. Discussed offline, and for this change moving the flag from opendistro config to es setting

Reply:
The backward incompatibility occurs when the following happens:
Old node does not have the code checked in, and a request to update this feature flag is sent to this old node. This will cause cluster to fail all requests and requires manual intervention.